static/js/marker.js

```diff
@@ -43,7 +43,7 @@ var MarkerView = Backbone.View.extend({
 
         if (this.model.get("provider_code") == PROVIDER_CODE_UNITED_HATZALA) {
             this.marker.setIcon(this.getIcon());
-            this.marker.setTitle(this.getTitle('united'));
+            this.marker.setTitle(this.model.get("title"));
```

Comment:
Wouldn't it be better to fix `getTitle` instead? Apparently it has logic specifically to construct the title for `united` markers: https://github.com/omerxx/anyway/blob/471-united-title/static/js/marker.js#L166